src/uwtools/drivers/orog.py

```diff
@@ -62,8 +62,9 @@ def input_config_file(self):
                 "blat",
             ]
             inputs = " ".join([str(inputs[i]) for i in ordered_entries])
-        outgrid = self.config["grid_file"]
-        orogfile = self.config.get("orog_file")
+        outgrid = "'{}'".format(self.config["grid_file"])
+        if orogfile := self.config.get("orog_file"):
+            orogfile = "'{}'".format(orogfile)
```

Comment:
These quotes are required or the program does not run correctly. It does not complain, but does produce garbage output.